pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -375,16 +401,17 @@ func evictPods(
 			if success {
 				klog.V(3).InfoS("Evicted pods", "pod", klog.KObj(pod), "err", err)
 
-				cpuQuantity := utils.GetResourceRequestQuantity(pod, v1.ResourceCPU)
-				nodeUsage.usage[v1.ResourceCPU].Sub(cpuQuantity)
-				totalAvailableUsage[v1.ResourceCPU].Sub(cpuQuantity)
-
-				memoryQuantity := utils.GetResourceRequestQuantity(pod, v1.ResourceMemory)
-				nodeUsage.usage[v1.ResourceMemory].Sub(memoryQuantity)
-				totalAvailableUsage[v1.ResourceMemory].Sub(memoryQuantity)
-
-				nodeUsage.usage[v1.ResourcePods].Sub(*resource.NewQuantity(1, resource.DecimalSI))
-				totalAvailableUsage[v1.ResourcePods].Sub(*resource.NewQuantity(1, resource.DecimalSI))
+				for name := range totalAvailableUsage {
+					switch name {
```

Comment:
if this only has 2 options, `Pods` or `default`, does it need to be a switch or could it just be an if/else?